lib/wrapper.js

```diff
@@ -85,11 +85,23 @@ if (typeof argv.m === 'string'){
 argv.f = p.resolve(argv.f);
 
 // Hack to force the wrapper process to stay open by launching a ghost socket server
-var server = net.createServer().listen();
+// Some security penetration tests create numerous daily socket errors that restart the wrapper.
+// Handling errors here is preferable to restarting the service for no real reason.
+function createKeepAliveServer() {
+  var sv = net.createServer((c) => {
+    c.on('error', (err) => {
+      log.warn(`Socket error (${err.code}) in wrapper keep-alive. Ignoring...`);
+    });
+  });
+  sv.listen();
+  return sv;
+}
+var server = createKeepAliveServer();
 
 server.on('error', function (err) {
-    launch('warn', err.message);
-    server = net.createServer().listen();
+  server.close(); // don't leak a trail of unclosed servers
```

Comment:
Hi @ChrisMiami,
I have a question. Does the subsequent servers also close?

The first server will close the socket because we are listening the event, but after the first error, we are not listening the error events for the new created servers.

Thank you very much for your contribution